docs/source/Usage.rst

```diff
@@ -49,18 +49,22 @@ with ``source venv/bin/activate`` to use FABulous.
     (venv)$ pip install -r requirements.txt
 
 
+Install FABulous with "editable" option:
+.. code-block:: console
+
+    (venv)$ pip install -e .
+
 The following packages need to be installed for the CAD toolchain
 
+If you are using yosys oss-cad-suite, no need to install the following packages.
```

Comment:
I would suggest to merge this sentence with the previous one to something like:

The following packages need to be installed for the CAD toolchain if you are not using the yosys oss-cad-suite.